diff -ru a/CMakeLists.txt systemsettings-4.10.5/CMakeLists.txt
--- a/CMakeLists.txt	2013-11-03 19:42:54.000000000 +0400
+++ systemsettings-4.10.5/CMakeLists.txt	2013-11-03 19:45:15.451238156 +0400
@@ -19,7 +19,7 @@
 find_package(KDE4 ${KDE_MIN_VERSION} REQUIRED)
 include(KDE4Defaults)
 
-find_package(Strigi REQUIRED)
+#find_package(Strigi REQUIRED)
 find_package(ZLIB REQUIRED)
 find_package(KActivities 6.0.0 )
 find_package(DBusMenuQt 0.6.0 REQUIRED)
diff -ru a/kcontrol/kfontinst/CMakeLists.txt systemsettings-4.10.5/kcontrol/kfontinst/CMakeLists.txt
--- a/kcontrol/kfontinst/CMakeLists.txt	2013-11-03 19:42:54.000000000 +0400
+++ systemsettings-4.10.5/kcontrol/kfontinst/CMakeLists.txt	2013-11-03 19:45:49.162671537 +0400
@@ -4,7 +4,7 @@
     find_package(Qt4 REQUIRED)
     find_package(Freetype REQUIRED)
     find_package(Fontconfig REQUIRED)
-    find_package(Strigi REQUIRED)
+#    find_package(Strigi REQUIRED)
     find_package(X11 REQUIRED)
     include(KDE4Defaults)
 endif (KFI_BUILD_STANDALONE)
@@ -39,7 +39,7 @@
     add_subdirectory( lib ) 
     add_subdirectory( dbus )
     add_subdirectory( kcmfontinst ) 
-    add_subdirectory( strigi-analyzer )
+#    add_subdirectory( strigi-analyzer )
     add_subdirectory( apps )
     add_subdirectory( kio ) 
     add_subdirectory( thumbnail )
