diff -ru a/CMakeLists.txt kdelibs-4.10.5/CMakeLists.txt
--- a/CMakeLists.txt	2013-11-03 23:27:50.170812241 +0400
+++ kdelibs-4.10.5/CMakeLists.txt	2013-11-03 22:56:01.089500109 +0400
@@ -74,15 +74,15 @@
 macro_log_feature(ZLIB_FOUND "ZLib" "Support for gzip compressed files and data streams" "http://www.zlib.net" TRUE "" "Required by the core KDE libraries and some critical kioslaves")
 
 set(STRIGI_MIN_VERSION 0.6.3)
-find_package(Strigi)
-if (WIN32)
-    set (STRIGI_REQUIRED FALSE)
-    set (STRIGI_EXTRA_TEXT "")
-else (WIN32)
-    set (STRIGI_REQUIRED TRUE)
-    set (STRIGI_EXTRA_TEXT "Required by some critical kioslaves")
-endif (WIN32)
-macro_log_feature(STRIGI_FOUND "Strigi" "Desktop indexing and search support" "http://strigi.sourceforge.net" ${STRIGI_REQUIRED} "${STRIGI_MIN_VERSION}" "${STRIGI_EXTRA_TEXT}")
+#find_package(Strigi)
+#if (WIN32)
+#    set (STRIGI_REQUIRED FALSE)
+#    set (STRIGI_EXTRA_TEXT "")
+#else (WIN32)
+#    set (STRIGI_REQUIRED TRUE)
+#    set (STRIGI_EXTRA_TEXT "Required by some critical kioslaves")
+#endif (WIN32)
+#macro_log_feature(STRIGI_FOUND "Strigi" "Desktop indexing and search support" "http://strigi.sourceforge.net" ${STRIGI_REQUIRED} "${STRIGI_MIN_VERSION}" "${STRIGI_EXTRA_TEXT}")
 
 set(LIBATTICA_MIN_VERSION "0.1.90")
 find_package(LibAttica)
diff -ru a/kio/CMakeLists.txt kdelibs-4.10.5/kio/CMakeLists.txt
--- a/kio/CMakeLists.txt	2013-11-03 22:54:04.000000000 +0400
+++ kdelibs-4.10.5/kio/CMakeLists.txt	2013-11-04 00:43:24.484313313 +0400
@@ -9,9 +9,9 @@
    set(KIO_NO_NEPOMUK TRUE)
 endif(NOT HAVE_NEPOMUK)
 
-if(NOT STRIGI_FOUND)
+#if(NOT STRIGI_FOUND)
    set(KIO_NO_STRIGI TRUE)
-endif(NOT STRIGI_FOUND)
+#endif(NOT STRIGI_FOUND)
 
 if(NOT SOPRANO_FOUND)
    set(KIO_NO_SOPRANO TRUE)
@@ -72,9 +72,11 @@
 add_subdirectory( kssl/kcm )
 add_subdirectory( misc )
 add_subdirectory( tests )
-if(STRIGI_FOUND)
+#if(STRIGI_FOUND)
+if(NOT KIO_NO_STRIGI)
    add_subdirectory( kio/dummyanalyzers )
-endif(STRIGI_FOUND)
+#endif(STRIGI_FOUND)
+endif()
 
 set(kiocore_STAT_SRCS
   kio/accessmanager.cpp
diff -ru a/kio/kio/kfilemetainfo.cpp kdelibs-4.10.5/kio/kio/kfilemetainfo.cpp
--- a/kio/kio/kfilemetainfo.cpp	2013-06-28 21:03:41.000000000 +0400
+++ kdelibs-4.10.5/kio/kio/kfilemetainfo.cpp	2013-11-04 00:40:33.726992606 +0400
@@ -421,9 +421,9 @@
 {
 }
 
-const KFileMetaInfo& KFileMetaInfo::operator= ( KFileMetaInfo const & kfmi )
+KFileMetaInfo& KFileMetaInfo::operator= ( KFileMetaInfo const & kfmi )
 {
-    return kfmi;
+    return *this;
 }
 
 KFileMetaInfo::~KFileMetaInfo()
